example/kotlin/settings.gradle

```diff
@@ -1,4 +1,5 @@
-apply from: file("../../node_modules/@react-native-community/cli-platform-android/native_modules.gradle");
-applyNativeModulesSettingsGradle(settings, "../..")
+rootProject.name = 'BrownfieldExample'
+apply from: file("../../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
```

Comment:
[nit]
```suggestion
apply from: file("../../node_modules/@react-native-community/cli-platform-android/native_modules.gradle")
applyNativeModulesSettingsGradle(settings)
```

Reply:
Is this some template generated `gradle.settings`? If so, please ignore the comment.